docs/getting_started/setup.rst

```diff
@@ -70,6 +70,9 @@ This file will pull in anything else that's required.
             <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
             <title>WYMeditor Quickstart</title>
             <script type="text/javascript" src="jquery/jquery.js"></script>
+            <!--for IE7 & IE8, ES5 shims are required-->
```

Comment:
We should use [conditional comments](http://www.quirksmode.org/css/condcom.html) to only include them for Less than IE10.